test/specs/welcome.js

```diff
@@ -0,0 +1,44 @@
+/* global describe, it, browser */
+const assert = require('chai').assert
+const fetch = require('node-fetch')
+
+browser.timeouts('script', 120000)
+
+const filterInstance = (instances, domain) => {
+  return instances.find(instance => {
+    return instance.doc && instance.doc.domain === domain
+  }) || instances[0]
+}
+
+const fetchRegisterToken = () => fetch(
```

Comment:
An idea for later improvement : It is possible to make a direct request to couchdb, with a JS client / ORM ? 
This syntax is not readable, it could be clearer to make a request to get the token, something like
`RegisterToken = couchdb.find(domain='cozy.local').RegisterToken`